src/stan-users-guide/style-guide.Rmd

```diff
@@ -314,74 +319,98 @@ layout.
 
 ### Line breaks between statements and declarations {-}
 
-It is dispreferred to have multiple statements or declarations on the
-same line, as in the following example.
+Each statement of a program should appear on its own line.
+Declaring multiple variables of the same type can be
+accomplished in a single statement with the syntax
 
 ```
-transformed parameters {
-  real mu_centered;  real sigma;
-  mu = (mu_raw - mean_mu_raw);    sigma = pow(tau,-2);
-}
+real mu, sigma;
 ```
 
-These should be broken into four separate lines.
-
 ### No tabs {-}
 
-Stan programs should not contain tab characters.  They are legal and
-may be used anywhere other whitespace occurs.  Using tabs to layout a
+Stan programs should not contain tab characters. Using tabs to layout a
 program is highly unportable because the number of spaces
 represented by a single tab character varies depending on which
 program is doing the rendering and how it is configured.
 
 ### Two-character indents {-}
 
 Stan has standardized on two space characters of indentation, which is
-the standard convention for C/C++ code.  Another sensible choice is
-four spaces, which is the convention for Java and Python.  Just be
-consistent.
+the standard convention for C/C++ code.
 
 ### Space between `if`, `{` and condition
 
-Use a space after `if`s.  For instance, use `if (x < y) ...`, not
-`if(x < y) ...`.
+Use a space after `if`s.  For instance, use `if (x < y) {...`, not
+`if(x < y){ ...`.
 
-### No space For function calls {-}
+### No space for function calls {-}
 
-There is no space between a function name and the function it applies
-to.  For instance, use `normal(0,1)`, not `normal (0,1)`.
+There should not be space between a function name and the arguments 
+it applies to.  For instance, use `normal(0, 1)`, not `normal (0,1)`.
 
 ### Spaces around operators {-}
 
 There should be spaces around binary operators.  For instance, use
 `y[1] = x`, not `y[1]=x`, use `(x + y) * z` not
 `(x+y)*z`.
 
+Unary operators are written without a space, such as in `-x`, `!y`.
+
+### No spaces in type constraints {-}
+Another exception to the above rule is when the operator is the 
+assignment operator `=` used inside a type constraint, such as
```

Comment:
"when the operator is the assignment operator" -> "when the assignment operator"